site/src/components/image.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Image
+description: Image component in the Atlas Design System
+template: standard
+---
+
+# Image
+
+The image component parents an image tag or svg and provides width based classes to constrain its children.
+
+## Usage
+
+The default image class takes up the full width of its container, but various sizes and screensizes modifiers are available.
+
+| Class pattern                | Sizes                              | screensize          |
+| ---------------------------- | ---------------------------------- | ------------------- |
+| `.image.image-<size>x<size>` | 16 24 32 36 48 64 76 88 96 112 128 | `tablet`, `desktop` |
+
+```html
+<div class="image">
+	<img src="~/src/scaffold/media/video-media.png" />
```

Comment:
Add `alt=""` to the examples so there are no red issues highlights in the browser